kernel/dw1000.c

```diff
@@ -1422,6 +1420,16 @@ static void dw1000_timestamp(struct dw1000 *dw,
 	/* Get 5-byte timestamp */
 	cc = le64_to_cpu(time->cc) & DW1000_TIMESTAMP_MASK;
 
+	/* Check it is really a new value */
+	if (unlikely(cc == dw->rx_timestamp)) {
+		dev_warn_ratelimited(dw->dev, "STAMP SKIP: RX timestamp "
+				     "repetition detected!\n");
+		return;
+	}
```

Comment:
Minor point, but could the "STAMP SKIP" become "timestamp skipped" or "ignoring timestamp", for consistency with the other driver messages?

Reply:
What's the justification for this change?  The bits being masked out are not just the DSP-generated bits, so I assume this is intended as a window check, in which case subtraction would be more appropriate than masking.